packages/studio-ui/src/web/views/FlowBuilder/inspector/index.tsx

```diff
@@ -108,19 +107,6 @@ class Inspector extends Component<Props> {
       )
     }
 
-    if (nodeType === 'say_something') {
-      return (
-        <SaySomethingForm
-          onDeleteSelectedElements={onDeleteSelectedElements}
-          contentType={currentFlowNode.content?.contentType}
-          formData={currentFlowNode.content?.formData}
-          updateNode={updateNodeAndRefresh}
-          readOnly={readOnly}
-          subflows={subflows}
-        />
-      )
-    }
```

Comment:
Do we get rid of it? You can actually use that node in the current version of Botpress by setting "experimental" to true in your botpress config file. 
![image](https://user-images.githubusercontent.com/42552874/153219997-a49f3127-7bc1-497c-b898-7d37509ee088.png)

Reply:
Ok we'll decide today what we do with those. Either we make em enabled by default or we remove all of them. I'll change the pr to only topics stuff

Reply:
@allardy ✅ 
Brought back the say something node to ensure non breaking change. We'll probably get rid of all content form in 13 and get rid of the experimental thing